lib/management/types.ts

```diff
@@ -262,18 +262,13 @@ export type AuditRecord = {
   data: Record<string, any>;
 };
 
-export enum UserStatus {
-  enabled = 'enabled',
-  disabled = 'disabled',
-  invited = 'invited',
-}
-
-export enum AuthzNodeExpressionType {
-  self = 'self', // direct relation expression
-  targetSet = 'targetSet', // expression via another relation definition target
-  relationLeft = 'relationLeft', // expression via parent relation like org within org and membership
-  relationRight = 'relationRight', // expression via child relation like folder within folder and owner
-}
+export type UserStatus = 'enabled' | 'disabled' | 'invited';
```

Comment:
This scares me for backward compatibility?

Reply:
it wasn't exported outside the package